lib/rabbit_feed/event_definitions.rb

```diff
@@ -44,6 +45,7 @@ def payload_contains &block
       end
 
       def field name, options
+        @sensitive_fields << name.to_s if options.delete(:sensitive)
         fields << (Field.new name, options[:type], options[:definition])
```

Comment:
Actually the reader is just that. For reading the data. This modifies it whilst preventing people from changing sensitive fields from outside the class. Now I can change it to `attr_accessor` but I don't think it's the right choice